test/e2e/components/vl-accordion.js

```diff
@@ -30,11 +26,20 @@ class VlAccordion extends VlElement {
     }
 
     async isOpen() {
-        return (await this.shadowRoot.findElement(By.css('#accordion-content'))).isDisplayed();
+        return (await this._content()).isDisplayed();
     }
     
     async isClosed() {
-        return !(await (await this.shadowRoot.findElement(By.css('#accordion-content'))).isDisplayed());
+        return !(await this.isOpen());
+    }
+
+    async contentSlotElements() {
+        const slottedContent = await (await this._content()).findElement(By.css("#accordion-slot"));
+        const slottedElements = await this.driver.executeScript('return arguments[0].assignedElements()', slottedContent);
+        const slottedVlElements = slottedElements.map((slot) => {
+            return new VlElement(this.driver, slot);
+        });
```

Comment:
Niet zo belangrijk (en is ook vooral stijl), maar kunt dit ook korter schrijven als `const slottedVlElements = slottedElements.map(slot => new VlElement(this.driver, slot));`.